src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/UntilSuccessfulRule.groovy

```diff
@@ -0,0 +1,10 @@
+package com.avioconsulting.mule.linter.rule.configuration
+
+class UntilSuccessfulRule extends ComponentCountRule {
+    static final String RULE_VIOLATION_MESSAGE = ' should be avoided if possible'
+
+    UntilSuccessfulRule() {
+        super('UNTIL_SUCCESSFUL_RULE', 'Until Successful Rule', "until-successful",
+                "http://www.mulesoft.org/schema/mule/core", 1)
```

Comment:
The core namespace should either be referenced from ConfigurationFile or we should move these into a Namespaces file as constants we can reference and not have to know or repeat.

Reply:
I'm all for creating a namespaces class